### DISCUSSION TITLE
How to Fix SMTP Error: Server Does Not Support Authentication

### DISCUSSION THREAD

**Discussion by BERKAY-YILDIRIM** [Mailu/Mailu#3610] [User support]
Hello everyone,
When I try to send an email using SMTP, I get the error: "ERROR: The SMTP server does not support authentication." I couldn't find anything on how to solve this issue. Does anyone have any information or suggestions on how I can fix this?

**Comment by ghostwheel42** [comment]
The SMTP port is used when receiving emails, not sending. You need to use the submission port (465) to send emails as shown in the client setup page of your server (see: https://test.mailu.io/admin/client)

**Reply by BERKAY-YILDIRIM** [reply]
![mailu](https://github.com/user-attachments/assets/c9cf30aa-44d5-4b4e-9e34-313e1456e3c6)

"The 465 port is not working, and since I am using Nginx Proxy Manager, I didn’t want to deal with SSL again, so I need to use port 25 instead."

**Reply by ghostwheel42** [reply]
When using non-TLS submission internally it is on port 587
